mssql/base.py

```diff
@@ -83,6 +83,9 @@ def encode_value(v):
         return '{%s}' % (v.replace('}', '}}'),)
     return v
 
+def handle_sql_variant_as_string(value):
+    # SQL variant of type 150 is not supported, convert it to string and return
+    return value.decode('utf-16le')
```

Comment:
I'm not too familiar with the sql_variant type but it looks like this method of decoding is likely to result in incorrect data.

Since sql_variant can contain multiple types, unpacking the data in a reliable manner seems difficult... We might be better off doing an explicit CAST in sql.

Reply:
We can know the base type of the field https://learn.microsoft.com/en-us/sql/t-sql/functions/sql-variant-property-transact-sql?view=sql-server-ver16

in the converter function, shall we convert whatever type to string and return to provide a uniform behaviour?

Reply:
@dauinsight, please let me know the best way to proceed. Thanks.